app/models/item.rb

```diff
@@ -30,9 +32,8 @@ class Item < ActiveRecord::Base
   # StateMachine has Issue with rails 4.2, it does not set initial
   # state by default
   # refer - https://github.com/pluginaweek/state_machine/issues/334
-  after_initialize :set_initial_state
-  before_save :set_description
-  after_commit :update_stockit_item, on: :update, unless: "GoodcitySync.request_from_stockit"
+  # after_initialize :set_initial_state
```

Comment:
Not setting initial state any more? @bharat619

Reply:
The `state_machine` gem had a tiny compatible issue with rails 4.2 where it does not set initial state. I did not find this on updating to rails 5

Reply:
Great, can you remove the callback and `set_initial_state` method?